docs/evaluate/temporal-cloud/billing-and-cost.mdx

```diff
@@ -0,0 +1,77 @@
+---
+id: billing-and-cost
+title: Billing and Cost
+sidebar_label: Billing and cost
+sidebar_position: 7
+description: As an Account Owner, you can access and manage billing details anytime, with invoices available for download on the Billing page. Your billing cycle starts on the 1st of each month, and manual payments are optional if auto-payment is set up.
+slug: /cloud/billing-and-cost
+toc_max_heading_level: 4
+keywords:
+  - billing
+  - invoices
+  - cost management
+  - account owner
+  - billing cycle
+  - payment options
+  - credit usage
+  - temporal cloud
+  - account balance
+  - invoice history
+tags:
+  - billing
+  - invoices
+  - cost management
+  - account owner
+  - billing cycle
+  - payment options
+  - credit usage
+  - temporal cloud
+  - account balance
+  - invoice history
+---
+
+As an Account Owner, you can view billing and cost details at any time. 
+
+The typical billing cycle begins on the first of the month (UTC). Your first month is prorated based on when you sign up. 
+
+You have the ability to download your Invoices by clicking the download icon by the date. 
+
+## About the Billing page
+
+The **Billing** page includes the following information:
+
+- Your current balance
+- Recent bill
+- Invoice history
+- (If applicable) Credit Grants
+- (If applicable) Credit Purchase
+- (If applicable) Credit Usage
+- (If applicable) Credits Remaining
+
+If you are not on a standard plan, the Billing page may not include all of the information stated above. 
+
+## How to navigate to the Billing page
+
+To navigate to your invoices, go to **Settings > Billing**
+
+## Current balance
+
+The current balance card displays the balance accrued thus far in the billing cycle. You will see an updated date of when this data is updated. Note, this information will change based on usage. This data also corresponds to the first line in the Invoices table. 
+
+## Recent bill
+
+The recent bill card display the last bill amount. If you are paying invoices through the Stripe portal, you will see a **Pay Now** button. The **Pay Now** button will take you to the Stripe portal for you to complete your payment. If the account set up for auto-payment, no action is required to pay the bill manually but it’s available if needed. 
```

Comment:
"if the account set up for auto-payment" --> "if the account is set up for auto-payment"